data/boarding missions.txt

```diff
@@ -34,6 +371,163 @@ mission "Assisting Hai"
 	on offer
 		payment 30000
 		conversation
-			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
+			`Knowing that you endangered your life to save the <origin>, the captain grants you <payment>.`
+				decline
+	destination Hai-home
+
+mission "Assisting Hai (Gifts)"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Hai
+	on offer
+		payment 30000
+		conversation
+			`Knowing you endangered your life to save the <origin>, the captain gives you <payment> along with a message reading, "Any bringer of gifts is welcome in Hai Space."`
+				decline
+	destination Hai-home
+
+mission "Assisting Hai (Threat)"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Hai
+	on offer
+		payment 30000
+		conversation
+			`Knowing you endangered your life to save the <origin>, the captain gives you <payment> along with a message reading, "Thank you for promoting peace and cooperation!."`
 				decline
 	destination Hai-home
+	
+mission "Assisting Unfettered Hai (Thief)"
```

Comment:
Not sure how I feel about this one, let alone how big of a reduction in cash it is. Also to note, if the player has less than 300,000 credits on them, then 0 credits will be taken and this conversation will make no sense.